### DISCUSSION TITLE
Opschalingsanalyse vraag 5: Wat zijn de verwachte effecten van het sensorenregister?

### DISCUSSION THREAD

**Discussion by marcvanandel** [kadaster-labs/sensrnet-home#256] [Pilot Gemeente]
Opschalingsanalyse vraag 5: **Wat zijn de verwachte effecten van het sensorenregister?**

**Comment by WOostdam** [comment]
- Meer bewustzijn, verantwoordelijkheid, volwassenheid over het gebruik van sensoren
- Inzicht in de aanwezige sensoren en daarmee in de aanwezige datastreams: kansen op breder gebruik van data, delen van data met derde partijen (indien toegestaan)

**Reply by marcvanandel** [reply]
En bewust zijn. O ja, en bewust zijn

**Comment by mikegroels** [comment]
- Real-time transparantie voor informatiezoekenden (i.t.t. het moeten stellen van een vraag aan de Gemeente, verzameltijd, wachten en vervolgens antwoord krijgen)
- Inzicht in sensorgebruik in de openbare ruimte geeft input voor het plaatsen voor nieuwe sensoren (door Gemeente en anderen), zodat verrommeling van die ruimte beperkt kan worden
- Beter asset management bij Gemeentes (en andere instanties) - Wat hebben we eigenlijk?
